.bazeliskrc

```diff
@@ -1 +1 @@
-USE_BAZEL_VERSION=5.4.1
+USE_BAZEL_VERSION=6.5.0
```

Comment:
Can you reason me through whether it's safe to do this given that our monorepo that submodules `stout` still uses `5.4.1`?

Reply:
I doubt that we can just point the mono repo at this commit and expect it to work. I'd suggest we (eventually) land this PR but keep the mono repo pointing at the previous commit, and then update the mono repo to point at this commit as part of the other work we need to do on Bazel 6.

Reply:
That's slightly risky, since we may need to come into `stout` and make bugfixes, so we'd essentially end up with two divergent `main`s. 

Perhaps a slightly safer approach: keep this PR open until right before we're ready to move the ~mountain~ monorepo to Bazel 6, then rebase it onto whatever the latest `main` is; the checks will tell us whether we're breaking anything.

Reply:
@rjhuijsman Sure, I'm fine not landing this PR until we're ready to move the mono repo to Bazel 6. Although I think the odds of us needing a bugfix in stout are pretty slim, so probably not a huge deal either way.

Reply:
OK, so we think Bazel 6 in `mono` is close enough to land this? Let's do it! 🎉